src/main/java/uk/gov/register/presentation/resource/Pagination.java

```diff
@@ -4,13 +4,15 @@
 import javax.ws.rs.BadRequestException;
 import java.util.Optional;
 
-class Pagination {
+public class Pagination {
 
     private final long pageIndex;
+    private final String resourcePath;
     private final long totalEntries;
     private final long pageSize;
 
-    Pagination(Optional<Long> pageIndex, Optional<Long> pageSize, long totalEntries) {
+    Pagination(String resourcePath, Optional<Long> pageIndex, Optional<Long> pageSize, long totalEntries) {
```

Comment:
Pagination class is used for a request so the number of entries is snapshot at the time of this request. no of entries are calculated for every request. 
Estimated entries are never greater than the total entries in table, so given the way it is used, it should never fail.